opencog/query/CMakeLists.txt

```diff
@@ -1,3 +1,30 @@
+IF (HAVE_GEARMAN)
+IF (HAVE_GUILE)
```

Comment:
Oops sorry, I misread that; ignore previous commment. Can you change to to HAVE_GUILE AND HAVE_GEARMAN ?